docs/spec_draft.md

```diff
@@ -2215,6 +2216,33 @@ implementation-defined value.
 
 [Back to Ops](#index-of-ops)
 
+## stablehlo.replica_id
+
+### Semantics
+
+Returns the unique id of the replica in the interval [0, N), where N is the
+number of replicas. Since all the replicas are running the same program, a
+`stablehlo.replica_id` call in the program will return a different value on each
+replica.
+
```

Comment:
Returns `replica_id` of a replica[#replicaexplanation_link]. Since all replicas run the same program, `stablehlo.replica_id` produces a different value on each replica.

We can omit the second sentence because across partitions replica_id might be same, also all of that should be clear from the parallel execution semantics.

Reply:
I would propose "Produces \`replica_id\` of the current process", because `replica_id` is already well-defined above, so there's no need to go into details of what a replica is or what is the range of what this op returns.

Agreed about explicitly referring to `replica_id` and about dropping the second sentence, given the discussion in #634. 

Re: linking to an explanation of what a replica_id is, I would say that this is not needed. E.g. we also don't explain what a token type, we just use it as needed.